src/pages/VideoPlayer/VideoPlayer.page.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { useLocation } from 'react-router';
+import Navbar from '../../components/Navbar';
+import SideMenu from '../../components/SideMenu/SideMenu.component';
+import VideoVisualizer from '../../components/VideoVisualizer/VideoVisualizer.component';
+import useVideo from '../../utils/hooks/useVideo';
+
+function VideoPlayer({ theme, setTheme }) {
+  const relatedId = new URLSearchParams(useLocation().search).get('v');
```

Comment:
it is good practice to grab hooks in a single line.

```suggestion
  const {search} = useLocation();
  const relatedId = new URLSearchParams(search).get('v');
```